lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,194 @@
 // lms - views - shopping cart
 // ====================
 
+$enrollment-details-width-difference: $baseline*11.85;
```

Comment:
@martynjames agreed to help with setting it up, since I'm having trouble SSHing into it

Will let you know as soon as it's up